### PR TITLE
b11

### PR DESCRIPTION
**Pull request by vinniefalco**
Odds and ends, clears up several issues

Reviewers: @seelabs @mellery451

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ project (Beast)
 set_property (GLOBAL PROPERTY USE_FOLDERS ON)
 
 if (WIN32)
-    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP /W4 /wd4100 /D_SCL_SECURE_NO_WARNINGS=1 /D_CRT_SECURE_NO_WARNINGS=1")
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP /W4 /wd4100 /bigobj /D _WIN32_WINNT=0x0600 /D_SCL_SECURE_NO_WARNINGS=1 /D_CRT_SECURE_NO_WARNINGS=1")
```
Comment (seelabs) [inline CMakeLists.txt]: The jamfile uses 0x0601 for this constant. We should use the same value for both build systems.

**Comment by mellery451** [comment]
looks good to me 👍

File: doc/http.qbk [hunk 2]
```diff
@@ -213,7 +209,7 @@ An asynchronous interface is available:
 ```
     void handle_write(boost::system::error_code);
     ...
-    http::request<http::empty_body> req;
+    http::request_v1<http::empty_body> req;
```
Comment (seelabs) [inline doc/http.qbk]: Typo on line 221 of this file: `paramter` should be `parameter`

File: test/websocket/websocket_async_echo_server.hpp [hunk 3]
```diff
@@ -94,7 +94,7 @@ class async_echo_peer
             stream<socket_type> ws;
             boost::asio::io_service::strand strand;
             opcode op;
-            beast::streambuf sb;
+            beast::streambuf db;
```
Comment (seelabs) [inline test/websocket/websocket_async_echo_server.hpp]: Did you mean to rename this one?

**Comment by vinniefalco** [comment]
Addressed review commits and force-pushed

**Comment by seelabs** [comment]
:+1:

**Comment by vinniefalco** [comment]
Waiting on CI

File: test/websocket/websocket_async_echo_server.hpp (hunk 3 above)
Comment (vinniefalco) [inline test/websocket/websocket_async_echo_server.hpp]: Will fix

File: doc/http.qbk (hunk 2 above)
Comment (vinniefalco) [inline doc/http.qbk]: Will fix

**Pull request merged**